qa/tasks/cephfs/test_volumes.py

```diff
@@ -6518,7 +6518,11 @@ def test_subvolume_snapshot_clone_quota_exceeded(self):
         self._fs_cmd("subvolume", "create", self.volname, subvolume,"--mode=777", "--size", str(osize))
 
         # do IO, write 50 files of 1MB each to exceed quota. This mostly succeeds as quota enforcement takes time.
-        self._do_subvolume_io(subvolume, number_of_files=50)
+        try:
+            self._do_subvolume_io(subvolume, number_of_files=50)
+        except CommandFailedError:
+            # command failed due to Disk quota exceeded as quota enforcement takes some.
```

Comment:
We should also change the comment here. As commented at line 6520, the testcase expects quota to be exceeded. So the current comment is confusing. We can just say

```
# ignore quota enforcement error.
```